ts/input/tex/textmacros/TextMacrosConfiguration.ts

```diff
@@ -39,6 +39,7 @@ import './TextMacrosMappings.js';
  */
 export const TextBaseConfiguration = Configuration.create('text-base', {
   parser: 'text',
+  priority: 1,
```

Comment:
Is there a reason for the priority?

Reply:
It was so that the base configuration is done first and then others that override anything in it don't need to specify their own priorities.  But I think that the packages may be processed in reverse order, which means I should make this 10 instead.

Reply:
Just checked, and this does mean text-base will have its `init()` and `config()` methods called before the other configurations.  So that means the default priority of 5 will put others after this, without having to set the priority.